app/src/js/multimocks.js

```diff
@@ -48,7 +48,8 @@ angular
     'multimocksData',
     function ($q, $http, $httpBackend, multimocksData) {
       var setupHttpBackendForMockResource = function (deferred, mock) {
-        var mockHeaders = multimocksData.getHeaders();
+        var mockHeaders = multimocksData.getHeaders(),
+          uriRegExp = new RegExp('^' + mock.uri + '$');
 
```

Comment:
We did, but I began to feel that wouldn't make for a sensible default. It could cause surprises. It's easy to work around by wrapping the expression in .*. =